saleor/payment/gateways/adyen/tests/webhooks/test_handle_notifications.py

```diff
@@ -22,6 +23,8 @@
     webhook_not_implemented,
 )
 
+logger = logging.getLogger(__name__)
```

Comment:
Just to confirm.
Is it something required to capture the log from different module?

Reply:
It's required to capture logger warnings.